src/form/FormInput.js

```diff
@@ -14,13 +14,22 @@ import normalize from '../helpers/normalizeText';
 const { width } = Dimensions.get('window');
 
 class FormInput extends Component {
+  constructor(props) {
+    super(props);
+    
+    this.getRef = this.getRef.bind(this);
+    this.focus = this.focus.bind(this);
+    this.blur = this.blur.bind(this);
+  }
+  
+  getRef() {
+    return this.props.textInputRef ? this.refs[this.props.textInputRef] : this.input;
+  }
   focus() {
-    const ref = this.props.textInputRef;
-    this.refs[ref].focus();
+    getRef() && getRef().focus();
```

Comment:
It's just a failsafe to make sure the ref actually exists before calling focus on it. It has to be this.getRef indeed